Tests/NIOOpenSSLTests/OpenSSLIntegrationTest.swift

```diff
@@ -14,11 +14,17 @@
 
 import XCTest
 import CNIOOpenSSL
-@testable import NIO
+import NIO
 @testable import NIOOpenSSL
 import NIOTLS
 import class Foundation.Process
 
+#if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
```

Comment:
I believe this shouldn't be necessary as the `NIO` should export them, no?

Reply:
Oh yeah, good call.